0001-Add-the-camera-sensor-gc5035-to-support-ADL-M.patch

```diff
@@ -0,0 +1,2471 @@
+From d237dcb058184a2f0f9ee35badf05cb787779c31 Mon Sep 17 00:00:00 2001
+From: "liang.wang" <liang1.wang@intel.com>
+Date: Tue, 28 Mar 2023 10:56:33 +0800
+Subject: [PATCH] Add the camera sensor gc5035 to support ADL-M.
+
+The sensor driver was updated to support several features:
+1. Add acpi to support x86 platform
+2. Add some resolutions/control and more regs specifically for ADL-M.
+3. Add gpio pins supported by int3272 and power on the sensor.
+4. v4l2 support.
+5. Adapt to intel IPU6-drivers, like 3A.
+The sensor driver gc5035 is modified based on the following link:
+https://patchwork.kernel.org/project/linux-media/patch/20200902224813.14283-4-tfiga@chromium.org
+
+Regarding to kernel update, don't forget to add the new sensor
+support in .config
+
+Signed-off-by: liang.wang <liang1.wang@intel.com>
+---
+ drivers/iommu/intel/iommu.c                   |   31 +
```

Comment:
We can contain the changes in int3472 only, and no need to include other changes since they have been in your source code.